beta/nncf/tensorflow/layers/wrapper.py

```diff
@@ -45,11 +45,14 @@ def __init__(self, layer, **kwargs):
                     input=layer.__class__.__name__))
 
         if 'name' not in kwargs:
-            kwargs['name'] = '{}_{}'.format('nncf_wrapper', layer.name)
+            kwargs['name'] = layer.name
 
         super().__init__(layer, **kwargs)
         self._track_trackable(layer, name='layer')
 
+        if 'name' not in kwargs:
```

Comment:
It looks like we can leave self.layer.name. Could you check it?

Reply:
I removed these changes and tests passed